core/src/main/java/io/grpc/internal/PickFirstLeafLoadBalancer.java

```diff
@@ -609,112 +611,165 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
     }
   }
 
-  /**
-   * Index as in 'i', the pointer to an entry. Not a "search index."
-   * All updates should be done in a synchronization context.
```

Comment:
Why delete this javadoc?

Reply:
It made no sense to me when I read it.  Rereading it I sort of get what it was going for, but it isn't really right since it contains the contents that can be pointed at in addition to the current location.  I'll try to come up with a more accurate replacement instead of just deleting it.